src/window/building/common.h

```diff
@@ -68,6 +68,8 @@ int window_building_get_vertical_offset(building_info_context *c, int new_window
 
 void window_building_draw_employment(building_info_context *c, int y_offset);
 
+void do_window_building_draw_employment(building_info_context* c, int y_offset, int consider_house_covering);
```

Comment:
Keep to the projects code conventions by prefixing the functions with the file they're in.

But more importantly: in this case I would say it's better to create a new function called `void window_building_draw_employment_without_house_cover(building_info_context *c, int y_offset);`
Then, in the `.c` file, create a new function `void draw_employment_info(building_info_context* c, int y_offset, int consider_house_covering);`, which is called by both public functions.

This way, the `consider_house_covering` flag is not exposed to the outside: this function is currently only called from the outside with the flag set to `0`.

Reply:
Ok it needed a little more code refactoring because we want to avoid duplicated code. But yes i did what you say here.